src/components/InfoCards/infoCards.css

```diff
@@ -0,0 +1,22 @@
+.infocard-container {
+  min-height: 540px;
+  width: 350px;
+  background: white;
+  border-radius: 5px;
+  margin: 15px;
+  padding: 15px;
```

Comment:
If we can we may want to use `rem` units for padding and margin so that in case we change the font size in the future the site or add in media queries for large displays the site will automatically adapt.